src/index.js

```diff
@@ -1,3 +1,48 @@
-// eslint-disable-next-line no-console
-console.log('Hello! The current date and time is: ' + Date());
+var fs;
+var promises;
+var request;
 
+const meetups = [
+  'Prov-JS',
+  'Kenzan-Providence-Hack-Nights',
+  'Girl-Develop-It-Providence',
+  'Tech-Collective-Meetups',
+  'MojoTech-Meetups',
+  'Rhode-Island-Makers-Microcontrollers-Robotics-Meetup',
+  'ux-meetup',
+  'IntraCityGeeks',
+  'WordPressRI'
+];
+
+request = require('request');
+fs = require('fs');
+
+function getData(url) {
+  return new Promise(function(resolve, reject) {
+
+    request(
+      { method: 'GET',
+        uri: url
+      }
+      , function (error, response, body) {
+
+        if (error) {
+          reject(error);
+        }
+        resolve(JSON.parse(body));
+      }
+    );
+  });
+
+}
+
+function init() {
+  promises = meetups.map(meetup => getData(`https://api.meetup.com/${meetup}/events`));
+
+  Promise.all(promises)
+    .then(results => {
+      fs.writeFile('meetup-data.json', JSON.stringify(results, null, 2));
```

Comment:
results at this point will be an array of event arrays structures somewhat like 
`[[{id:'23'}],[{id:'32'},{id:'13'}],[{id:'323',id:'483'}]]`
consider flattening this array so that it looks more like
`[{id:'23'},{id:'32'},{id:'13'},{id:'323',id:'483'}]`
before writing to the file.